tests/test_trade.py

```diff
@@ -78,6 +97,58 @@ def test_init_only(self):
         """Tests base LP setups"""
         self.run_base_trade_test(agent_policies=[], target_liquidity=1e6, target_pool_apr=0.05)
 
+    def test_compare_agent_to_calc_liquidity(self):
+        """Compare agent init as above to old calc_liquidity method"""
+        for target_liquidity in (1e2, 1e3, 1e4, 1e5, 1e6, 1e7, 1e8, 1e9):
+            for target_pool_apr in (0.01, 0.03, 0.05, 0.10, 0.25, 0.5, 1, 2):  # breaks at 500%
+                logging.debug(
+                    "running test with target_liquidity=%g and target_pool_apr=%g",
+                    target_liquidity,
+                    target_pool_apr,
+                )
+                simulator = self.run_base_trade_test(
```

Comment:
Took me a bit to realize what is going on here -- can you add a comment that explains how you're using this? e.g. `# run_base_trade_test initializes a market with an LP agent`